packages/dd-trace/src/debugger/index.js

```diff
@@ -24,13 +25,19 @@ function start (config, rc) {
   configChannel = new MessageChannel()
 
   rc.setProductHandler('LIVE_DEBUGGING', (action, conf, id, ack) => {
-    const ackId = `${id}-${conf.version}`
-    rcAckCallbacks.set(ackId, ack)
+    rcAckCallbacks.set(++ackId, ack)
```

Comment:
In theory you'd have to worry about running into MAX_SAFE_INTEGER, but I think even this happens once per milisecond, you'll have centuries of leeway.